go-controller/pkg/libovsdbops/switch.go

```diff
@@ -224,11 +227,16 @@ func createOrUpdateLogicalSwitchPortsOps(nbClient libovsdbclient.Client, ops []l
 	for i := range lsps {
 		lsp := lsps[i]
 		opModel := operationModel{
-			Model:          lsp,
-			OnModelUpdates: onModelUpdatesAll(),
-			DoAfter:        func() { sw.Ports = append(sw.Ports, lsp.UUID) },
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model: lsp,
+			OnModelUpdates: []interface{}{
+				&lsp.Addresses,
+				&lsp.Type,
+				&lsp.TagRequest,
+				&lsp.Options,
```

Comment:
Missing port security, ExternalIDs.

With that, this is equivalent as none of the fields go back to default from non-default.

Reply:
options is map